src/base/registrations/registrar.ts

```diff
@@ -75,22 +81,28 @@ export async function getRegistration(name: string, config: Config, resolver?: E
     resolver.getText('url'),
     resolver.getText('eth.radicle.seed.id'),
     resolver.getText('eth.radicle.seed.host'),
+    resolver.getText('eth.radicle.seed.git'),
+    resolver.getText('eth.radicle.seed.api'),
     resolver.getText('eth.radicle.anchors'),
     resolver.getText('com.twitter'),
     resolver.getText('com.github'),
   ]);
 
-  const [address, avatar, url, seedId, seedHost, anchorsAccount, twitter, github] =
-    meta.map(r => r.status == "fulfilled" ? r.value : null);
+  const [address, avatar, url, seedId, seedHost, seedGit, seedApi, anchorsAccount, twitter, github] =
+    meta.map(r => r.status == "fulfilled" ? r.value : undefined);
```

Comment:
I think we have an issue here since `r.value` can also be `null`, since `getText` returns `string | null`. Eventually we could do something like 
```suggestion
    meta.map(r => r.status == "fulfilled" ? r.value ?? undefined : undefined);
```